test/functions/math.test.cpp

```diff
@@ -173,3 +173,15 @@ TEST_CASE("pow with string base", "[functions][math][pow][string]") {
             "08862990907940498760592041482791966715057053380053636634477979263"
             "4601932346249911644260403539535769743430516736");
 }
+
+TEST_CASE("Sqrt of big integers", "[functions][math][sqrt][big]") {
+    BigInt big_num1 = "3479680743635798438602889954485038919559460388450";
+    REQUIRE(sqrt(big_num1) == "1865390238967653234193278");
+
+    BigInt big_num2 = "471336517784520212796937741396686403167942977773121645803479339647737161";
+    REQUIRE(sqrt(big_num2) == "686539523832765372419381874316283619");
+
+    BigInt big_num3 = "304953007708768585645827645127232631942308013843965557450677671087391006548030000";
+    REQUIRE(sqrt(big_num3) == "17462903759362833573945713140457169848646");
```

Comment:
* Please add at least 5 more cases with numbers of different magnitudes - such as 5 digits, 20 digits, 100 digits, 500 digits etc.
* Verify base cases in a separate test case (as in case of `pow()`). The base cases will check the square root of 0, 1 and -1.